Makefile

```diff
@@ -1,6 +1,8 @@
 .PHONY: clean test lint init docs build-docker build-docker-full build-docker-mitie-en build-docker-spacy-en build-docker-spacy-de
 
 JOBS ?= 1
+INTEGRATION_TEST_FOLDER = tests/integration_tests/
+PYTEST_MARKERS ?= "sequential or not sequential"
```

Comment:
`PYTEST_MARKERS` feels a bit generic to me, maybe include integration in the name?